back/engines/commercial/public_api/app/models/public_api/policy_wrapped_phase.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# This is used to solve Pundits limitation of only allowing one policy per model
+# https://github.com/elabs/pundit/issues/241
+
+module PublicApi
+  class PolicyWrappedPhase < SimpleDelegator
+    def self.policy_class
+      PublicApi::PhasePolicy
```

Comment:
Consistency: in phase_policy.rb you use the namespace resolution operator (`::`) as a prefix, but here you don't.
I think the module name `PublicApi` is not needed, because this code lives inside the module `PublicApi`.
So I guess this would be good:
```suggestion
      PhasePolicy
```

Reply:
You're right. Just in this case I'd rather keep the prefix. Not for the machine, but for the human. If you don't know the codebase well, you might not know there is a `PublicApi::PhasePolicy`, and assume it gets resolved to `::PhasePolicy`

Reply:
> Not for the machine, but for the human

Very good.